usr/src/cmd/fs.d/smbclnt/smbutil/view.c

```diff
@@ -86,15 +84,6 @@ cmd_view(int argc, char *argv[])
 	while ((opt = getopt(argc, argv, STDPARAM_OPT)) != EOF) {
 		if (opt == '?')
 			view_usage();
-		/*
-		 * This is an undocumented option, just for testing.
-		 * Use the old LanMan Remote API Protocol (RAP) for
-		 * enumerating shares.
-		 */
-		if (opt == 'B') {
```

Comment:
I take it this is ancient stuff that we no longer want.

Reply:
Correct.  Modern servers no longer support the Remote Administration Protocol (RAP),
which is the ancient predecessor to RPC over named pipes.